src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -135,9 +168,27 @@ export default {
 			// We wait for 1.5 times the animation length to give the animation time to really finish.
 			}, 1.5 * animationLength)
 		},
+
 		toggleNavigationByEventBus({ open }) {
 			this.toggleNavigation(open)
 		},
+
+		/**
+		 * Activate focus trap if it is currently needed, otherwise deactivate
+		 */
+		toggleFocusTrap() {
+			if (this.isMobile && this.open) {
+				this.focusTrap.activate()
+			} else {
+				this.focusTrap.deactivate()
+			}
+		},
+
+		handleEsc() {
+			if (this.isMobile) {
+				this.toggleNavigation(false)
```

Comment:
This does not seem to work currently in my testing

Reply:
the reason was probably that I tested on a screen with higher res than 1024px (so ismobile was false)